vnext/PropertySheets/React.Cpp.props

```diff
@@ -17,6 +17,7 @@
 
   <PropertyGroup>
     <USE_HERMES>false</USE_HERMES>
+    <USE_V8>true</USE_V8>
```

Comment:
If you define these like this
```
 <USE_HERMES Condition="'$(USE_HERMES)' == ''">false</USE_HERMES>
<USE_V8 Condition="'$(USE_V8)' == ''">false</USE_V8>
```
it is much easier for consumers to override the setting, either with environment variable or by passing in the msbuild arguments.

Reply:
Also probably want to make the default false for USE_V8

Reply:
Done..